images/kindnetd/cmd/kindnetd/main.go

```diff
@@ -70,11 +70,9 @@ func main() {
 	// enforce ip masquerade rules
 	// TODO: dual stack...?
 	masqAgent := NewIPMasqAgent(net.IsIPv6String(hostIP), []string{os.Getenv("POD_SUBNET")})
+	// TODO: handle the errors and logging here, temporally the agent will run forever
```

Comment:
Temporally?
Also we are not handling errors here? but the logging point applies to the sync forever code.

Reply:
I was trying to understand what happens if the goroutine dies (somehow) or if it is constantly failing, shouldn't we handle it from here?

Reply:
Goroutines dont just die.
If it was constantly failing that should be handled inside the method, possibly with a panic.